test/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+if(NOT TARGET gtest OR NOT TARGET gmock)
+  # Prevent overriding the parent project's compiler/linker settings on Windows
+  set(gtest_force_shared_crt # cmake-lint: disable=C0103
+      ON
+      CACHE BOOL "" FORCE)
+  add_subdirectory(
+    "${PROJECT_SOURCE_DIR}/extern/qcec/extern/qfr/extern/dd_package/extern/googletest"
+    "extern/qcec/extern/qfr/extern/dd_package/extern/googletest" EXCLUDE_FROM_ALL)
+  add_subdirectory(
+          "${PROJECT_SOURCE_DIR}/extern/ddsim/extern/qfr/extern/dd_package/extern/googletest"
+          "extern/ddsim/extern/qfr/extern/dd_package/extern/googletest" EXCLUDE_FROM_ALL)
+endif()
+
+package_add_test(
+  ${PROJECT_NAME}_test
+  ${PROJECT_NAME})
```

Comment:
this probably doesn't work out of the box because you need to have at least one source file (in the test directory) for this macro to work correctly.

Reply:
Done in https://github.com/tyi1025/dd_eval/pull/1/commits/143bb197854bfda8eb4056f9b0b3c0e4c07b7998
Although I commented these lines out instead of removing them because I figured we might need them later anyway. Please let me know if removing is the better choice here.

Reply:
That's reasonable for now! 👍🏻